tidb-cloud/changefeed-sink-to-mysql.md

```diff
@@ -78,7 +78,7 @@ The **Sink to MySQL** connector can only sink incremental data from your TiDB cl
 
 After completing the prerequisites, you can sink your data to MySQL.
 
-1. Navigate to the **Changefeed** tab of your TiDB cluster.
+1. Navigate to the **Data Replication** tab of your TiDB cluster and click the **Changefeed** tab.
```

Comment:
this might conflict with https://github.com/pingcap/docs/pull/11515/files#diff-c95401c1c761cca89172f4f336be2ef3421dd1c9d83e635d8fb8d8fae3372b35